docs/navigation-parameters-reference.md

```diff
@@ -27,7 +27,12 @@ Luigi.setConfig({
         children: [node, node, node],
         hideFromNav: false,
         isolateView: false,
-        icon: 'settings'
+        icon: 'settings',
+        category: {
+          label: 'General',
+          icon: 'general'
+        }, 
```

Comment:
Maybe a comment with an 'or'?
```
category: {
  label: 'General',
  icon: 'general'
},//OR 
category: 'General'
```

Reply:
Yea, also thought about this, but we didn't have such case until now. Will add one

Reply:
done